pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "jupyter-repo2docker"
+dynamic = ["version"]
+description = "repo2docker: Turn code repositories into Jupyter enabled Docker Images"
+readme = "README.md"
+license = "BSD-3-Clause"
+authors = [
+    {name = "Project Jupyter Contributors", email = "jupyter@googlegroups.com"}
+]
+maintainers = [
+    {name = "Project Jupyter Contributors", email = "jupyter@googlegroups.com"}
+]
+keywords = ["reproducible", "science", "environments", "docker"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "Intended Audience :: Science/Research",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Software Development",
+    "Topic :: System :: Systems Administration",
+]
+requires-python = ">=3.9"
```

Comment:
Should we change, **in a separate pull request**, the CI job to run the tests with Python 3.9 so that we don't introduce code that only run in Python 3.10 or higher?

Reply:
We [have that already](https://github.com/jupyterhub/repo2docker/blob/2456a4dc917f36248b6128907299570e94f87e7f/.github/workflows/test.yml#L72-L75).

But we should probably also require 3.10, soon.